packages/react/src/components/UserDropdownMenu/index.ts

```diff
@@ -16,6 +16,7 @@
  * under the License.
  */
 
-export {default} from './ButtonDropdownMenu';
-export {ButtonDropdownMenuProps} from './ButtonDropdownMenu';
-export {ModeListInterface} from './ButtonDropdownMenu';
+export {default} from './UserDropdownMenu';
+export {UserDropdownMenuProps} from './UserDropdownMenu';
+export {ModeListInterface} from './UserDropdownMenu';
```

Comment:
Lets ditch the Interface suffix.

Reply:
Fixed https://github.com/wso2/oxygen-ui/pull/21/commits/59365a5711b8bf4b43fb857ef2b95c8446611b76